test_helpers/clean_feature_context.go

```diff
@@ -49,6 +90,74 @@ func CleanFeatureContext(s *godog.Suite) {
 		return killTestContainers(dockerComposeDir, appName)
 	})
 
+	s.Step(`^my machine has running application and test containers$`, func() error {
+		appContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", appNetwork, "up")
+		appContainerProcess.SetDir(path.Join(appDir, "tmp"))
+		err := appContainerProcess.Start()
+		if err != nil {
+			return err
+		}
+		testContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", testNetwork, "up")
+		testContainerProcess.SetDir(path.Join(appDir, "service", "tests", "tmp"))
+		err = testContainerProcess.Start()
+		time.Sleep(2500 * time.Millisecond)
+		return err
+	})
+
+	s.Step(`^my machine has stopped application and test containers$`, func() error {
+		appContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", appNetwork, "create")
+		appContainerProcess.SetDir(path.Join(appDir, "tmp"))
+		err := appContainerProcess.Start()
+		if err != nil {
+			return err
+		}
+		testContainerProcess = execplus.NewCmdPlus("docker-compose", "-p", testNetwork, "create")
+		testContainerProcess.SetDir(path.Join(appDir, "service", "tests", "tmp"))
+		return testContainerProcess.Start()
```

Comment:
This and the previous step seep quite similar with the only difference being "up" and "create". Can we dry that up?